### PR TITLE
Dw 667 costs scaffold

### PR DESCRIPTION
**Pull request by DanielLyleWilliamWarren**
Scaffolded area for Cost Effectiveness Scenarios and added routing.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -13,5 +13,28 @@
...
+    "statusBarItem.remoteBackground": "#12a362",
+    "statusBarItem.remoteForeground": "#000000",
+    "tab.activeBorder": "#17d17e",
+    "titleBar.activeBackground": "#12a362",
+    "titleBar.activeForeground": "#000000",
+    "titleBar.border": "#12a362",
+    "titleBar.inactiveBackground": "#12a36299",
+    "titleBar.inactiveForeground": "#00000099"
+  },
+  "peacock.color": "#12a362"
```
Comment (bgsandan) [inline .vscode/settings.json:39]: I can just see the ghost of @jon571 'tsk'ing at you :)

**Comment by DanielLyleWilliamWarren** [comment]
Ah, removed peacock for now... but I changed the code so it'll need a rereview @bgsandan !

**Review by bgsandan — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (jon571) [reply .vscode/settings.json]: tsk.
Less of the GHOST you!

Comment (Fuhji) [reply .vscode/settings.json]: 👻

**Pull request merged**